app/assets/javascripts/application.js.erb

```diff
@@ -389,11 +389,13 @@ var Lobsters = new _Lobsters();
 
 $(document).ready(function() {
   var $olcomments = $("ol.comments");
-  $olcomments.on("click", ".comment a.downvoter", function() {
+  $olcomments.on("click", ".comment .voters > a + div + a", function() {
```

Comment:
These kind of selectors seem very fragile to me. Actually, looking at the HTML I'm not even sure if I fully understand what this is matching exactly. Is it not possible to keep the simple `a.downvoter` match?

Reply:
I had to add an additional container to the page to get the desired result of having a bigger tap area and out of a personal preference I did not assign a class to this container. The `.comment .voters > a + div + a` just matches on an `a` that is located after an `a` and a `div` on the first level of the voters `div`. I do not expect that the voters container will see much changes in the future. But you not immediately groking the selector shows that it is too complicated so I'm gonna add a class to the outer container and clean up the selector.